ProvisionSample/Program.cs

```diff
@@ -295,6 +329,40 @@ static void Main(string[] args)
             }
         }
 
+        static async Task<Import> GetStatus(string workspaceCollectionName, string workspaceId, string importId)
```

Comment:
Not a big deal, but could we name this GetImport instead, as it's returning the full Import object? That would also be less ambiguous if we added new methods later that got the status of other types.